app/controllers/api/v1/books_controller.rb

```diff
@@ -5,17 +5,16 @@ module V1
     # BooksController
     class BooksController < ApplicationController
       def index
-        render json: Book.all.order(id: :asc)
+        @books = Book.paginate(page: params[:page], per_page: 10).order(id: :asc)
```

Comment:
To simplify our method calls in the controller (remember, thin controller) we can transfer this long query into the `Book.model`. You can probably create a class method there:

```ruby
# in /models/book.rb:

self.paginate_results(page)
 Book.paginate(page: page, per_page: 10).order(id: :asc)
# also probably it's better to show books in descending order that they were created
# meaning books that were added last should appear first
end
```

And then using it like this in the controller:

```ruby
# /controllers/api/v1/books_controller.rb

def index
  @books = Book.paginate_results(params[:page])
end
```